include/mbedtls/debug.h

```diff
@@ -304,6 +310,12 @@ void mbedtls_debug_printf_ecdh( const mbedtls_ssl_context *ssl, int level,
                                 mbedtls_debug_ecdh_attr attr );
 #endif
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3) && defined(MBEDTLS_SSL_SESSION_TICKETS)
+void mbedtls_debug_print_ticket_flags( const mbedtls_ssl_context *ssl, int level,
```

Comment:
```suggestion
void mbedtls_debug_print_ticket_flags(
              const mbedtls_ssl_context *ssl, int level,
```
Vertical alligned, and this line exceeds 80 characters.